tests/controller/templates_manager/local_temp/container.py

```diff
@@ -0,0 +1,45 @@
+from jumpscale import j
+from zerorobot.service_collection import ServiceNotFoundError
+from testconfig import config
+import random
+
+class CONTManager:
+    def __init__(self, parent, service_name=None):
+        self.container_template = 'github.com/threefoldtech/0-templates/container/0.0.1'
+        self._parent = parent
+        self.logger = self._parent.logger
+        self.robot = self._parent.remote_robot
+        self._container_service = None
+        if service_name:
+            try:
+                self._container_service = self.robot.service.get(name=service_name)
+            except ServiceNotFoundError:
```

Comment:
raise the error cause we used the class with `service_name` and we can't find it